### DISCUSSION TITLE
Bug in Firefly III Graphics and Navigation menus

### DISCUSSION THREAD

**Discussion by Halberto** [firefly-iii/firefly-iii#9422] [Firefly III]
### Support guidelines

- [x] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [x] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [ ] ... there are no open or closed issues that are related to my problem
- [x] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I am having an issue with graphs not showing up. I am pushing my instance through a reverse proxy. When I go directly to the IP locally, I can see all the graphs, and the buttons and menus work fine, but when I access the instance remotely from my subdomain, all the graphs show a warning sign, and some buttons and menus do not work. Would you happen to have any ideas on how to fix this?

### Debug information

![Image](https://github.com/user-attachments/assets/d1e12033-e4be-4244-8d89-5427cd07de81)

**Comment by JC5** [comment]
Hi, thanks for opening a discussion. 

This looks like a configuration error. Could you please share any errors you see in the browser console?

**Comment by Halberto** [comment]
![image](https://github.com/user-attachments/assets/dfb573c0-5886-4ccf-9b56-45f071af5c9a)

**Reply by JC5** [reply]
Great, now you know what the error is. The search bar is in the top right corner of your browser. Let me know if you have any questions.

**Reply by Halberto** [reply]
I have already solved issue thank you.
